src/braket/circuits/gates.py

```diff
@@ -3300,6 +3301,123 @@ def gpi(
 Gate.register_gate(GPi)
 
 
+class PhaseRx(DoubleAngledGate):
+    r"""Phase Rx gate.
+
+    Unitary matrix:
+
+        .. math:: \mathtt{PhaseRx}(\theta,\phi) = \begin{bmatrix}
+                cos(\theta / 2) & -i e^{-i \phi} sin(\theta / 2) \\
+                -i e^{i \phi} sin(\theta / 2) & cos(\theta / 2)
+            \end{bmatrix}.
+
+    Args:
+        angle_1 (Union[FreeParameterExpression, float]): The first angle of the gate in
+            radians or expression representation.
+        angle_2 (Union[FreeParameterExpression, float]): The second angle of the gate in
+            radians or expression representation.
+    """
+
+    def __init__(
+        self,
+        angle_1: Union[FreeParameterExpression, float],
+        angle_2: Union[FreeParameterExpression, float],
+    ):
+        super().__init__(
+            angle_1=angle_1,
+            angle_2=angle_2,
+            qubit_count=None,
+            ascii_symbols=[_multi_angled_ascii_characters("PhaseRx", angle_1, angle_2)],
+        )
+
+    def to_matrix(self) -> np.ndarray:
+        r"""Returns a matrix representation of this gate.
+
+        Returns:
+            np.ndarray: The matrix representation of this gate.
+        """
+        _theta = self.angle_1
+        _phi = self.angle_2
+        return np.array(
+            [
+                [
+                    np.cos(_theta / 2),
+                    -1j * np.exp(-1j * _phi) * np.sin(_theta / 2),
+                ],
+                [
+                    -1j * np.exp(1j * _phi) * np.sin(_theta / 2),
+                    np.cos(_theta / 2),
+                ],
+            ]
+        )
+
+    def _qasm_name(self) -> str:
```

Comment:
There was a discussion that the class is named "PRx", and the method name "prx". I will link the discussion to you.